flyteadmin/pkg/manager/impl/project_manager.go

```diff
@@ -127,6 +127,16 @@ func (m *ProjectManager) UpdateProject(ctx context.Context, projectUpdate admin.
 	return &response, nil
 }
 
+func (m *ProjectManager) GetProject(ctx context.Context, request admin.ProjectGetRequest) (*admin.Project, error) {
+	projectModel, err := m.db.ProjectRepo().Get(ctx, request.Id)
```

Comment:
nit: can we add validation that ID is not empty since it's a required param?
in the same vein as https://github.com/flyteorg/flyte/blob/master/flyteadmin/pkg/manager/impl/validation/project_validator.go#L35

Reply:
@katrogan nice catch, I've also added a test case for it

Reply:
thank you!

Reply:
also so sorry to be so nit-picky, but can we keep the validation with the other commands in https://github.com/flyteorg/flyte/blob/master/flyteadmin/pkg/manager/impl/validation/project_validator.go ?

Reply:
Keep them coming! Done.